src/test/ui/dep-graph/dep-graph-trait-impl-two-traits.stderr

```diff
@@ -1,4 +1,4 @@
-error: no path from `x::<impl Foo for char>` to `typeck`
```

Comment:
@cjgillot the modifications in `compiler/rustc_typeck/src/coherence/inherent_impls.rs` caused this change.

Reply:
The current code checks each module independently. Having those checks performed on the whole crate at once create new dependency edges. This may be what causes the diagnostic changes.

Reply:
This can be resolved by adding a `eval_always` to the `hir_crate_items` query declaration.

Reply:
`hir_crate_items` already has `eval_always`.

```rust
    /// All items in the crate.
    query hir_crate_items(_: ()) -> rustc_middle::hir::ModuleItems {
        storage(ArenaCacheSelector<'tcx>)
        eval_always
        desc { "get HIR crate items" }
    }
```